pages/apis/rest_api/pipeline_templates.md

```diff
@@ -0,0 +1,293 @@
+    "name": "Sam Kim",
+    "email": "sam@example.com",
+    "avatar_url": "https://www.gravatar.com/avatar/example",
+    "created_at": "2013-05-03T04:17:55.867Z"
+  }
+}
+```
+
+Required scope: `read_pipeline_templates`
+
+Success response: `200 OK`
+
+## Create a pipeline template
+
+```bash
+curl -X POST "https://api.buildkite.com/v2/organizations/{org.slug}/pipeline-templates" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "name": ":hammer: Build",
+    "description": "Shared build steps configuration",
+    "configuration": "steps:\n  - label: \":hammer: Build\"\n    command: \"scripts/build.sh\"",
+    "available": true,
+  }'
+```
+
+```json
+{
+  "uuid": "018a86cc-db73-7d15-8c68-5023cf8d64c3",
+  "graphql_id": "UGlwZWxpbmVUZW1wbGF0ZS0tLTAxOGE4NmNjLWRiNzMtN2QxNS04YzY4LTUwMjNjZjhkNjRjMw==",
+  "name": "Build template",
+  "description": "Shared build steps configuration",
+  "configuration": "steps:\n  - label: \":hammer: Build\"\n    command: \"scripts/build.sh\"",
+  "available": true,
+  "url": "http:///api.buildkite.com/v2/organizations/acme-inc/pipeline-templates/018a86cc-db73-7d15-8c68-5023cf8d64c3",
+  "web_url": "http://www.buildkite.com/organizations/acme-inc/pipeline-templates/018a86cc-db73-7d15-8c68-5023cf8d64c3",
+  "created_at": "2023-05-03T04:17:55.867Z",
+  "created_by": {
+    "id": "3d3c3bf0-7d58-4afe-8fe7-b3017d5504de",
+    "graphql_id": "VXNlci0tLTNkM2MzYmYwLTdkNTgtNGFmZS04ZmU3LWIzMDE3ZDU1MDRkZQo=",
+    "name": "Sam Kim",
+    "email": "sam@example.com",
+    "avatar_url": "https://www.gravatar.com/avatar/example",
+    "created_at": "2023-08-29T10:10:03.000Z"
+  },
+  "updated_at": "2023-06-12T04:17:55.867Z",
+  "updated_by": {
+    "id": "3d3c3bf0-7d58-4afe-8fe7-b3017d5504de",
+    "graphql_id": "VXNlci0tLTNkM2MzYmYwLTdkNTgtNGFmZS04ZmU3LWIzMDE3ZDU1MDRkZQo=",
+    "name": "Sam Kim",
+    "email": "sam@example.com",
+    "avatar_url": "https://www.gravatar.com/avatar/example",
+    "created_at": "2013-05-03T04:17:55.867Z"
+  }
+}
+```
+
+Required [request body properties](/docs/api#request-body-properties):
+
+<table class="responsive-table">
+<tbody>
+  <tr>
+    <th><code>name</code></th>
+    <td>Name for the pipeline template.<br><em>Example:</em> <code>"Build template"</code></td>
+  </tr>
+  <tr>
+    <th><code>configuration</code></th>
+    <td>YAML step configuration for the pipeline template.<br><em>Example:</em> <code>"steps:\n  - command: "scripts/build.sh"</code></td>
+  </tr>
+</tbody>
+</table>
+
+Optional [request body properties](/docs/api#request-body-properties):
+
+<table class="responsive-table">
+<tbody>
+  <tr>
+    <th><code>description</code></th>
+    <td>Description for the pipeline template.<br><em>Example:</em> <code>"Shared build steps configuration"</code></td>
+  </tr>
+  <tr>
+    <th><code>available</code></th>
+    <td>When set to <code>true</code>, non-admins can assign the pipeline template can assigned to pipelines.<br><em>Example:</em> <code>false</code></td>
```

Comment:
Looks like this change didn't get applied quite right. Should be:

```suggestion
    <td>When set to <code>true</code>, non-admins can assign the pipeline template to pipelines.<br><em>Example:</em> <code>false</code></td>
```

I think this mistake is made in a few places.

Reply:
Hopefully that last commit I just pushed up covered the rest of it 🤞